packages/graph-retriever/src/graph_retriever/testing/adapter_tests.py

```diff
@@ -232,7 +209,21 @@ class GetAdjacentCase:
             "chinchilla",
         ],
     ),
-    # Add test for ID edges and metadata edges combined.
+    GetAdjacentCase(
+        "metadata_and_id",
+        "domesticated hunters",
+        edges={
+            IdEdge("boar"),
+            IdEdge("chinchilla"),
+            MetadataEdge("type", "mammal"),
+        },
+        expected=[
```

Comment:
One thing this test shows -- we may want to think about a way of "boosting" the score of some edges (possibly general weights). Here, it feels like an edge specifically mentioning a `boar` may be worth more than an edge saying "find mammals".

My guess is that "horse" and "llama" come back because they are highly relevant to `domesticated`, while `boar` and `chinchilla` are not.

Perhaps this is to be expected?

Although, I likely want to find a few other IDs to use in the search terms -- probably a non-mammal to include.